### PR TITLE
Update getstarted

### PR DESCRIPTION
**Pull request by minjikarin**
updated get started

### CODE REVIEW

File: docs/get-started/moving-to-production.md [hunk 1]
```diff
@@ -37,12 +37,27 @@ Once you have a production key, you make the following changes to your source co
...
-                 client_secret='YOUR_PRODUCTION_CLIENT_SECRET',
-                 hostname='production')
+If you are using [Amadeus for Developers SDKs](https://github.com/amadeus4dev), add `hostname='production'` in the client together with your API key and API secret as shown below example in [python SDK](https://github.com/amadeus4dev/amadeus-python):
+
+```python
+from amadeus import Client, ResponseError
+
+amadeus = Client(
+    client_id='REPLACE_BY_YOUR_API_KEY',
+    client_secret='REPLACE_BY_YOUR_API_SECRET'
```
Comment (tsolakoua) [inline docs/get-started/moving-to-production.md:47]: a comma is missing here

File: docs/get-started/first-api-call.md [hunk 2]
```diff
@@ -17,24 +19,29 @@ To start using the APIs you need to tell the system that you are allowed to do s
 !!! danger
     Remember that the API Key and API Secret are for personal use only. Do not share them with anyone.
 
-All you need to do is to attach an alphanumeric string called **token** to your calls. This token will identify you as valid user and is generated from two parameters: `API Key` and `API Secret`. In order to get them, and once your account has been verified, you can [sign in](https://developers.amadeus.com/signin) and follow these steps:
+All you need to do is to attach an alphanumeric string called **token** to your calls. This token will identify you as a valid user and is generated from two parameters: `API Key` and `API Secret`. To get them, and once your account has been verified, you can [sign in](https://developers.amadeus.com/signin) and follow these steps:
+
+* Click on your username \(top right corner\) 
+* Go to [My Self-Service Workspace](https://developers.amadeus.com/my-apps) 
+    ![api_key1](../images/api_key1.png)
```
Comment (tsolakoua) [inline docs/get-started/first-api-call.md:26]: this pic looks a bit blurry

**Review by tsolakoua — CHANGES_REQUESTED** [review]

File: docs/get-started/first-api-call.md [hunk 3]
```diff
@@ -17,24 +19,29 @@ To start using the APIs you need to tell the system that you are allowed to do s
...
 
-* Click on your username \(top right corner\)
-* Go to [My Self-Service Workspace](https://developers.amadeus.com/my-apps)
 * Click on **Create New App** button
+    ![api_key2](../images/api_key2.png)
```
[the remark this answers is not in the document]
Comment (minjikarin) [reply docs/get-started/first-api-call.md:29]: can't resolve it yet ... to be investigated further once we have more pics

File: docs/get-started/first-api-call.md (hunk 2 above)
Comment (minjikarin) [reply docs/get-started/first-api-call.md:26]: can't resolve it yet ... to be investigated further once we have more pics

**Pull request merged**